controllers/v1beta1/build_helpers.go

```diff
@@ -303,90 +303,6 @@ func (r *LagoonBuildReconciler) getOrCreateConfigMap(ctx context.Context, cmName
 	return nil
 }
 
-// getOrCreatePromoteSARoleBinding will create the rolebinding for openshift promotions to be used by the lagoon-deployer service account.
```

Comment:
This is unused since promotes are handled differently, theres no need for this in kubernetes deployments